### PR TITLE
fix(typo): fix typo of issue1053

### PR DESCRIPTION
**Pull request by jscode017**
Signed-off-by: jsjs026 <jasonhack518@gmail.com>

### CODE REVIEW

**Comment by jscode017** [comment]
related: #1053

**Review by takeutak — APPROVED** [review]

**Comment by takeutak** [comment]
@jscode017 Thanks to your commit! LGTM.

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 1]
```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
     eventFilter: object | null = null;
 
     startTransaction(req: Request, businessLogicID: string, tradeID: string): void {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:26]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 2]
```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
     }
 
     executeNextTransaction(txInfo: object, txId: string): void {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:30]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 3]
```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
     }
 
     getOperationStatus(tradeID: string): object {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:34]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 4]
```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
         return {};
     }
 
     setConfig(data: []): object {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:39]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 5]
```diff
@@ -23,25 +23,25 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
         return {};
     }
 
     onEvent(ledgerEvent: LedgerEvent, targetIndex: number): void {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:44]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 6]
```diff
@@ -56,17 +56,17 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
     }
 
     getEventDataNum(ledgerEvent: LedgerEvent): number {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:59]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 7]
```diff
@@ -56,17 +56,17 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
         return 0;
     }
 
     getTxIDFromEvent(ledgerEvent: LedgerEvent, targetIndex: number): string | null {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:64]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

File: packages/business-logic-plugin/BusinessLogicBase.ts [hunk 8]
```diff
@@ -56,17 +56,17 @@ export class BusinessLogicBase implements BusinessLogicPlugin {
...
         return null;
     }
 
     hasTxIDInTransactions(txID: string): boolean {
-        // NOTE: This method implements the BisinessLogcPlugin operation(* Override by subclass)
+        // NOTE: This method implements the BusinessLogcPlugin operation(* Override by subclass)
```
Comment (petermetz) [inline packages/business-logic-plugin/BusinessLogicBase.ts:69]: @jscode017 Is this meant to be `BusinessLogicPlugin`?

**Review by petermetz — CHANGES_REQUESTED** [review]
@jscode017 Please update the commit message to be compliant with the formatting requirements of the semantic PR check. Let me know if any questions, happy to help!

File: packages/business-logic-plugin/BusinessLogicBase.ts (hunk 1 above)
Comment (jscode017) [reply packages/business-logic-plugin/BusinessLogicBase.ts]: Yeah sorry for that

**Comment by jscode017** [comment]
@petermetz, I think this is ok for review

**Review by petermetz — APPROVED** [review]

**Pull request merged**